tests/postgres_tests/test_citext.py

```diff
@@ -0,0 +1,31 @@
+"""
+Test cases for the citext extension from postgres.  It supports indexing of case-insensitive text strings
+and thus eliminates the need for operations such as iexact and other modifiers to enforce use of an index.
+"""
+from django.db import IntegrityError
+
+from . import PostgreSQLTestCase
+from .models import CitextTestModel
+
+
+class CitextTestCase(PostgreSQLTestCase):
+
+    def setUp(self):
```

Comment:
Don't you mean ``setUpTestData``? The tests fail with your suggestion, and in the other tests for ``django.contrib.postgres``, setUpTestData is used.